lib/stat_tracker.rb

```diff
@@ -301,34 +301,24 @@ def least_accurate_team(specific_season)
     # def most_tackles
 
     # end
-    
-    def fewest_tackles(specific_season)
-        specific_season_integer = specific_season.to_i
-        games_in_season = {specific_season => []}
-        @game_stats_data.each do |game_id, game_object|
-            games_in_season[specific_season].push(game_id) if game_object.season == specific_season_integer
-        end
 
-        team_total_tackles = {}
+    def team_tackles_in_games(game_ids)
+        team_total_tackles = Hash.new(0)
 
-        @seasons_stats_data.each do |game_key, game_object|
-            if games_in_season[specific_season].include?(game_object.game_id)
-                if !(team_total_tackles.keys.include?(game_object.team_id))
-                    team_total_tackles[game_object.team_id] = 0
-                    team_total_tackles[game_object.team_id] += game_object.tackles
-                else
-                    team_total_tackles[game_object.team_id] += game_object.tackles
-                end
-            end
-        end
-        lowest_tackling_team = team_total_tackles.min_by do |team_id, tackles|
-            tackles
+        @seasons_stats_data.each do |key, season_object|
```

Comment:
Love me a guard clause. Especially when it avoids an if statement in an if statement.